src/docs/content/docs/configuration/reaper_specific.md

```diff
@@ -0,0 +1,343 @@
+blocking in cause some thread is waiting for I/O, like calling a Cassandra cluster through JMX.
+
+</br>
+
+### `scheduleDaysBetween`
+
+Type: *Integer*
+
+Default: *7*
+
+Defines the amount of days to wait between scheduling new repairs. The value configured here is the default for new repair schedules, but you can also define it separately for each new schedule. Using value 0 for continuous repairs is also supported.
+
+</br>
+
+### `segmentCount`
+
+Type: *Integer*
+
+Defines the default amount of repair segments to create for newly registered Cassandra repair runs (token rings). When running a repair run by the Reaper, each segment is repaired separately by the Reaper process, until all the segments in a token ring are repaired. The count might be slightly off the defined value, as clusters residing in multiple data centers require additional small token ranges in addition to the expected. This value can be overwritten when executing a repair run via Reaper.
+
+</br>
+
+### `server`
+
+Settings to configure the application UI server.
+
+    server:
+      type: default
+      applicationConnectors:
+        - type: http
+          port: 8080
+          bindHost: 0.0.0.0
+      adminConnectors:
+        - type: http
+          port: 8081
+          bindHost: 0.0.0.0
+      requestLog:
+        appenders: []
+
+Reaper provides two separate UIs; the application UI which is configured via the `applicationConnectors` settings and the administration UI which is configured via the `adminConnectors` settings. The application UI provides functionality to create/manage cluster schedules, and the administration UI provides tools for monitoring and debugging the Reaper system.
+
+#### `port`
+
+For the `applicationConnectors` this setting will be the port number used to access the application UI. For the `adminConnectors` this setting will be the port number to access the administration UI.
+
+Note that the port numbers for each must be different values when bound to the same host.
+
+#### `bindHost`
+
+For the `applicationConnectors` this setting will be the host address used to access the application UI. For the `adminConnectors` this setting will be the host address used to access the administration UI.
+
+Note that to bind the service to all interfaces use value **0.0.0.0** or leave the value for the setting this blank. A value of **\*** is an invalid value for this setting.
+
+</br>
+
+### `storageType`
+
+Type: *String*
+
+Whether to use database or memory based storage for storing the system state. The value must be either **cassandra**, **database** or **memory**. If the recommended (persistent) storage type **database** or **cassandra** is being used, the database client parameters must be specified in the respective `database` or `cassandra` section in the configuration file. See the example settings in provided testing configuration in *src/test/resources/cassandra-reaper.yaml*.
```

Comment:
We should make the last part a link to the yaml file (*src/test/resources/cassandra-reaper.yaml*)

Reply:
I agree. The directory where the example configuration files point to has been changed as well.